src/hotspot/share/runtime/interfaceSupport.inline.hpp

```diff
@@ -192,6 +192,7 @@ class ThreadInVMfromUnknown {
 
 
 class ThreadInVMfromNative : public ThreadStateTransition {
+  ResetNoHandleMark __rnhm;
```

Comment:
We should add a constructor for RNHM that takes the current thread so that we don't have to call Thread::current() unnecessarily in the existing RNHM constructor.

Reply:
Ok.

Reply:
I forgot to push the change up to add a ResetNoHandleMark that takes a Thread* parameter.  Since it evaporates in product mode, I don't think it's needed or helpful.  I could add it back with https://github.com/openjdk/jdk/pull/1990 if you insist.